Formula/qp.rb

```diff
@@ -0,0 +1,24 @@
+class Qp < Formula
+  desc "The query-pipe: command-line (ND)JSON querying"
+  homepage "https://github.com/paybase/qp"
+  url "https://github.com/paybase/qp/archive/1.0.1.tar.gz"
+  sha256 "6ef12fd4494262899ee12cc1ac0361ec0dd7b67e29c6ac6899d1df21efc7642b"
+
+  depends_on "quickjs" => :build
+
+  def install
+    mkdir bin.to_s
+    system "curl", "https://unpkg.com/@paybase/csp@1.0.8/dist/esm/index.js", "--output", "./src/csp.js"
```

Comment:
This is not acceptable, please use a resource block.

Reply:
Thank you for your feedback, please check it again. I changed to use resource block.